.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+# Checklist for Pull Request
+🚨Please review the [guidelines for contributing](https://auto-wcag.github.io/auto-wcag/pages/contribute.html) to this repository.
+
+- [ ] Make sure you are requesting to **pull a issue/feature/bugfix branch** (right side). Don't request your master branch!
+- [ ] Make sure you are making a pull request against the **master branch** (left side). Also you should start *your branch* off *master branch*.
+- [ ] Kindly prefix the pull request name with `DRAFT` or `FINAL` accordingly. For more details, read [Rule Design Guidelines](https://auto-wcag.github.io/auto-wcag/pages/design/review.html)
+- [ ] Provide a reference to the issue being addressed.
+
```

Comment:
Could there be a note on naming conventions for files too? (e.g. draft-...)

Reply:
@annethyme - think I have already covered that by requesting to prefix PR with `DRAFT` or `FINAL`.